docs/serving/samples/tag-header-based-routing/README.md

```diff
@@ -0,0 +1,118 @@
+This sample illustrates the feature "Tag Header Based Routing".
+
+## Tag Header Based Routing Feature
+
+The feature "Tag Header Based Routing" allows users to send requests directly to specific tagged Revisions with
+the same URL of Knative Service. In order to achieve this, users only need to set the specific header "Knative-Serving-Tag:
+{revision-tag}" into the request.
+
+Currently Istio, Contour and Kourier Ingress support this feature.
+
+## Prerequestie
+
+1. A Knative cluster that has an Istio Ingress controller or Contour Ingress controller installed
+with Knative version 0.16 and above.
```

Comment:
```suggestion
1. A Knative cluster that has an ingress controller installed
with Knative version 0.16 and above.
```

Reply:
Doc already mentions which ingresses are supported, so I don't think we need to call them out again, or else also add Kourier here?

Reply:
SGTM